scripts/submit_skims_UL18.sh

```diff
@@ -1,512 +1,399 @@
-
-
-######################
-#### Reweighting ggF non res - filelists up to date
-### norm xs = 1 pb
-#python scripts/skimNtuple.lp.py -T $OUTDIR -s True -c  config/skim_Legacy2018.lp.cfg  -n 20 -q long -k True --pu $PUDIR/PU_Legacy2018_SF.txt -o $SKIMDIR/$OUTDIR/SKIM_HHRew_SM  -x 1.0 --kl 1.0  --kt 1.0 -x 1. -a True -i $INPUTDIR_SIG/GluGluToHHTo2B2Tau_LO_allNodes.txt
-
-#this is just for cross check with LO node SM, which wrongly has c2g set to 1
-#python scripts/skimNtuple.lp.py -T $OUTDIR -s True -c  config/skim_Legacy2018.lp.cfg  -n 20 -q long -k True --pu $PUDIR/PU_Legacy2018_SF.txt -o $SKIMDIR/$OUTDIR/SKIM_HHRew_SM_wrong  -x 1.0 --kl 1.0  --kt 1.0 --c2 0.0 --cg 0.0 --c2g 1. -a True -i $INPUTDIR_SIG/GluGluToHHTo2B2Tau_LO_allNodes.txt
+echo "-------- Run: $(date) ---------------" >> ${ERR_FILE}
+
+### Submission command
+function run_skim() {
+	comm="python ${KLUB_DIR}/${SUBMIT_SCRIPT} --tag ${TAG_DIR} -o ${OUTSKIM_DIR} -c ${KLUB_DIR}/${CFG} "
+	comm+="--exec_file ${EXEC_FILE} -q long -Y 2018 -k True --pu ${PU_DIR} $@"
+	[[ ${DRYRUN} -eq 1 ]] && echo ${comm} || ${comm}
+}
+
+### Input file list production command
+function produce_list() {
+	comm="python ${KLUB_DIR}/${LIST_SCRIPT} -t ${KLUB_TAG} --data_period ${DATA_PERIOD} $@"
+	[[ ${DRYRUN} -eq 1 ]] && echo ${comm} || ${comm}
+}
+
+### Extract sample full name
+function find_sample() {
+	nargs=$(( ${3}+3 ))
+	if [ $# -ne ${nargs} ]; then
+		echo "Wrong number of arguments - ${nargs} expected, $# provided"
+        exit 1
+    fi
+	pattern=${1}
+	list_dir=${2}
+	lists=${@:4}
+
+	sample=""
+	nmatches=0
+	for ldata in ${lists[@]}; do
+		[[ ${ldata} =~ ${pattern} ]] && { sample=${BASH_REMATCH[0]}; nmatches=$(( ${nmatches} + 1 )); }
+	done
+	if [ ${nmatches} -eq 0 ]; then
+		mes="The ${pattern} pattern was not found in ${list_dir} ."
+		echo ${mes} >> ${ERR_FILE}
+		echo ${mes}
+		return 1
+	elif [ ${nmatches} -gt 1 ]; then
+		mes="The ${pattern} pattern had ${nmatches} matches in ${list_dir} ."
+		echo ${mes} >> ${ERR_FILE}
+		echo ${mes}
+		return 1
+	fi
+	echo ${sample}
+}
+
+### Run on data samples
+# DATA_LIST=("EGamma" "Tau" "SingleMuon" "MET")
+# RUNS=("Run2018A" "Run2018B" "Run2018C" "Run2018D")
+DATA_LIST=("MET")
+RUNS=("Run2018A" "Run2018B")
+for ds in ${DATA_LIST[@]}; do
+	for run in ${RUNS[@]}; do
+		pattern="${ds}__${run}"
+		sample=$(find_sample ${pattern} ${LIST_DATA_DIR} ${#LISTS_DATA[@]} ${LISTS_DATA[@]})
+		if [[ ${sample} =~ ${SEARCH_SPACE} ]]; then
+			ERRORS+=( ${sample} )
+		else
+			[[ ${NO_LISTS} -eq 0 ]] && produce_list --kind Data --sample ${sample}
+		 	run_skim -n 90 --isdata True -i ${DATA_DIR} --sample ${sample}			
+		fi
+	done
+done
+exit 1
```

Comment:
that should be removed before merging, otherwise everything seem to behave as expected

Reply:
The line was introduced for testing. Should be fine now. Thank you for pointing it out.